packages/e2e-tests/test-applications/nextjs-app-dir/test-recipe.json

```diff
@@ -17,6 +17,11 @@
       "dependencyOverrides": {
         "next": "latest"
       }
+    },
+    {
+      "dependencyOverrides": {
+        "next": "canary"
```

Comment:
Do we need to have `latest` if we already have `canary`?

Reply:
Yes. They could theoretically diverge.